plugins/org.eclipse.sirius.diagram.sequence.ui/src/org/eclipse/sirius/diagram/sequence/ui/tool/internal/edit/policy/SequenceNodeCreationPolicy.java

```diff
@@ -163,11 +163,11 @@ protected Command getCreateNodeOnNodeCommand(CreateRequest request, NodeCreation
                 GraphicalHelper.logical2screen(bottomRight, (IGraphicalEditPart) getHost());
                 request.setSize(new Dimension(LayoutConstants.DEFAULT_EXECUTION_WIDTH, LayoutConstants.DEFAULT_EXECUTION_HEIGHT));
             }
-            CreationUtil creationUtil = new CreationUtil(request, getDiagramCommandFactory(startingEndPredecessor, startingEndPredecessor, location), getRealLocation(request), request.getSize(),
```

Comment:
As I see, it seems OK to remove the request because I assume that the result it's equivalent with `getRealLocation(request, parentEditPartToUse)`. Do you confirm ?

Reply:
I found it difficult to understand how the request was used before. With your changes, it's easier, but I wasn't sure I understood the “before” version.